kymatio/scattering1d/backend/jax_backend.py

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Are we including the Jax backend in this PR?

Reply:
My bad.